config/manager/manager.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: postgres-operator
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: operator
+        image: postgres-operator
+        env:
+        - name: CRUNCHY_DEBUG
+          value: "true"
+        - name: PGO_DISABLE_PGCLUSTER
+          value: "true"
```

Comment:
Any chance these can be booleans or no?

Reply:
These must be strings, otherwise the Deployment will be rejected by the API server, e.g.:

```bash
Error from server (BadRequest): error when creating "./config/default": Deployment in version "v1" cannot be handled as a Deployment: v1.Deployment.Spec: v1.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Env: []v1.EnvVar: v1.EnvVar.Value: ReadString: expects " or n, but found t, error found in #10 byte of ...|,"value":true},{"nam|..., bigger context ...|ainers":[{"env":[{"name":"CRUNCHY_DEBUG","value":true},{"name":"PGO_DISABLE_PGCLUSTER","value":true}|...
```